package.json

```diff
@@ -22,28 +23,28 @@
     "extend": "^3.0.2",
     "fast-xml-parser": "^4.1.3",
     "generic-pool": "^3.8.2",
-    "@google-cloud/storage": "^6.9.3",
     "glob": "^7.1.6",
     "https-proxy-agent": "^5.0.1",
     "jsonwebtoken": "^9.0.0",
     "mime-types": "^2.1.29",
     "mkdirp": "^1.0.3",
-    "mock-require": "^3.0.3",
     "moment": "^2.29.4",
     "moment-timezone": "^0.5.15",
     "open": "^7.3.1",
     "python-struct": "^1.1.3",
     "simple-lru-cache": "^0.0.2",
     "string-similarity": "^4.0.4",
-    "test-console": "^2.0.0",
     "tmp": "^0.2.1",
     "urllib": "^2.38.0",
     "uuid": "^3.3.2",
     "winston": "^3.1.0"
   },
   "devDependencies": {
+    "mocha": "^10.1.0",
+    "mock-require": "^3.0.3",
+    "nyc": "^15.1.0",
```

Comment:
when you are adding nyc you should write in README how to run it to gather code coverage, add script and add nyc output directory to git ignore - but maybe you should add nyc in separate PR?